src/app/components/Modal/index.tsx

```diff
@@ -0,0 +1,30 @@
+import MatModal from '@mui/material/Modal'
+import { ModalTypeMap } from '@mui/material/Modal/Modal'
+import { FC } from 'react'
+import { COLORS } from '../../../styles/theme/colors'
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+
+const ModalContent = styled(Box)(({ theme }) => ({
+  position: 'absolute',
+  width: '100%',
+  maxWidth: '700px',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%,-50%)',
+  padding: `${theme.spacing(4)} ${theme.spacing(4)}`,
+  boxShadow: '-20px 4px 40px rgba(34, 47, 63, 0.24)',
+  backgroundColor: COLORS.white,
+  borderRadius: theme.spacing(4),
+  [theme.breakpoints.up('sm')]: {
+    padding: `${theme.spacing(5)} ${theme.spacing(6)}`,
+  },
+}))
+
+export const Modal: FC<ModalTypeMap['props']> = ({ children, ...restProps }) => {
```

Comment:
Modal is not scrollable. Tho current use case fits on landscape phones, it isn't generalizable

![8aed7004 oasis-explorer pages dev_emerald_blocks](https://user-images.githubusercontent.com/3758846/230674219-8863d374-dc96-42fc-ac7b-b12ccc600882.png)